File: src/Models/Apartments.php
<?php

namespace EON\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Apartments extends Model
{
    use HasFactory;

    protected $table = 'apartments';

    protected $hidden = [
        'created_at',
        'updated_at',
    ];

    protected $fillable = [
        'uaid',
        's_total',
        's_living',
        's_kitchen',
        'height',
        'price',
        'floor',
    ];
}

File: internal/domain/Apartments.go
package domain

type Apartments struct {
	ID       string `xml:"id,attr"`
	STotal   string `xml:"s_total,attr"`
	SLiving  string `xml:"s_living,attr"`
	SKitchen string `xml:"s_kitchen,attr"`
	Height   uint64 `xml:"height,attr"`
	Price    string `xml:"price,attr"`
	Floor    uint64 `xml:"floor,attr,omitempty"`
}

type Data struct {
	Apartments []Apartments `xml:"apartments>apartment"`
}
